packages/react/src/primitives/SliderField/SliderField.tsx

```diff
@@ -64,7 +64,7 @@ const SliderFieldPrimitive: Primitive<SliderFieldProps, typeof Root> = (
   );
 
   const values = isControlled ? [value] : undefined;
-  const defaultValues = !isControlled ? [defaultValue] : undefined;
+  const defaultValues = !isControlled ? [defaultValue || 0] : undefined;
```

Comment:
If we set `defaultValue`'s default value to 0 at line 27 above, then we won't need this change or the one at line 82, since currentValue is derived from defaultValue or value.